source/com.microsoft.tfs.util/tests/com/microsoft/tfs/util/email/EmailAddressParserTest.java

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See License.txt in the repository root.
+
+package com.microsoft.tfs.util.email;
+
+import com.microsoft.tfs.util.StringUtil;
+
+import junit.framework.TestCase;
+
+public class EmailAddressParserTest extends TestCase {
+    final EmailAddressParser parser = new EmailAddressParser();
+
+    public void testEmailAddressGoodSimple() {
+        final boolean result = parser.parse("JohnDoe@live.com"); //$NON-NLS-1$
```

Comment:
Consider using [Reserved Top Level DNS Names](http://tools.ietf.org/html/rfc2606) in test code.